### DISCUSSION TITLE
Add support for reactions

### DISCUSSION THREAD

**Discussion by gilcu3** [d99kris/nchat#213] [Ideas]
It would be wonderful to support reactions, as both upstream libraries seem to support it already [whatsmeow](https://github.com/tulir/whatsmeow/issues/125) and [tdlib](https://github.com/tdlib/td/issues/1818). I am happy to help with the backend implementation but would need help on the frontend side, ui decisions are normally more tricky :)

**Comment by d99kris** [comment]
Hi @gilcu3 - thanks for offering help! Actually I've started (and almost completed) adding reactions support locally. I'm doing some final testing and target to integrate it this weekend if all goes well. I'll update here again once it's added.

**Comment by d99kris** [comment]
Hi - support for reactions has been added in 1a652d9. Select a message and press alt-s to add/change/remove reaction for a message. Reactions for messages are displayed below them, similar to mobile messaging apps. Note that color themes (except the default) have been updated to have a slightly different background color for reactions, so it may make sense to apply/copy themes again (or modify ones own accordingly).
